data/json/items/comestibles/egg.json

```diff
@@ -894,6 +894,45 @@
     "fun": 2,
     "flags": [ "FREEZERBURN" ]
   },
+  {
+    "id": "curing_roe",
+    "type": "COMESTIBLE",
+    "copy-from": "egg_fish",
+    "name": { "str_sp": "curing roe" },
+    "description": "Roe in the process of becoming fully cured with salt, capable of becoming homemade caviar.  The flavor is slowly developing at this stage, and is significantly unpleasant due to the residual salt.  Activate it once completed to make use of the final product.",
+    "weight": "296 g",
+    "volume": "500ml",
+    "calories": 68,
+    "price": 500,
+    "price_postapoc": 250,
+    "phase": "liquid",
+    "spoils_in": "7 days",
+    "fun": -15,
+    "use_action": {
+      "target": "post_apoc_caviar",
+      "msg": "You remove the roe from the brine and now have post apocalypse caviar.",
+      "moves": 150,
+      "type": "delayed_transform",
+      "transform_age": 1800,
+      "not_ready_msg": "The roe isn't fully cured yet.",
+      "//": "30 minutes"
+    }
+  },
+  {
+    "id": "cured_roe",
+    "type": "COMESTIBLE",
+    "copy-from": "egg_fish",
+    "name": { "str_sp": "post apocalyptic caviar" },
+    "description": "Fish eggs that have been cured in a salty brine.  Can be vacuum sealed to preserve freshness.  Technically caviar only came from a specific type of Sturgeon roe but the FDA allows all cured fish to be referred to as caviar.",
```

Comment:
If it would still be caviar pre-Cataclysm, why would it need this name 🤔

~~Also who cares what the FDA thinks when they ded~~

Reply:
I put it in the text so caviar purists can be pointed to it.  But I'm open to other suggestions for a name.

Reply:
Also it would be confusing to anyone outside of the US without the line possibly.

Reply:
How about putting caviar in quotes?